### DISCUSSION TITLE
Doks 1.2 is out!

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#1145] [Announcements]
Doks 1.2 is out and features support for diagrams, extended code blocks formatting, and more.

Diagrams are now available in 1.2. You can use a fenced code block to embed an SVG image of a diagram in your Doks site using the free [Kroki](https://kroki.io/) service.

Doks now extends the formatting possibilities for code blocks. Code blocks can be rendered inside a window-like frame. A frame that looks like a terminal window will be used for shell scripting languages (e.g. `bash` or `sh`). Other languages display inside a code editor-style frame if they include a title.

Read more in the [blog post announcement](https://getdoks.org/blog/doks-1-2/)

**Comment by jollydodo** [comment]
Awesome! Update went well and this immediately fixed the problems I was having with the default mermaid implementation. The only 'downside' I could find is that the diagrams right now don't respect dark-mode, but for me that is not a big deal.
